main.py

```
@@ -118,18 +118,18 @@ def __init__(self):
         for i in range(math.ceil(self.radiation / 100)):
             self.radiationbar += ">"
         
-        for i in range(11-len(list(self.radiationbar))):
+        for i in range(20-len(list(self.radiationbar))):
             self.radiationbar += "-"
 
-        self.radiationbar += "\U0001F480"
+        self.radiationbar += "X"
 
         # generates the hunger bar
         for i in range(math.ceil(self.hunger / 10)):
```

Ruff (0.11.9)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)